Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -860,6 +860,21 @@ void MainWindow::updateViewerBBox()
   qglviewer::Vec 
     vec_min(xmin, ymin, zmin),
     vec_max(xmax, ymax, zmax);
+  qglviewer::Vec offset(0,0,0);
+  for(int i=0; i<3; ++i)
+  {
+    if(log2(bbox.min(i)) > 13.0 )
```

Comment:
This line does not compile on Windows: you must use [`std::log2`] and include `<cmath>`.

[`std::log2`]: http://en.cppreference.com/w/cpp/numeric/math/log2